pkg/api/registered/registered.go

```diff
@@ -22,46 +22,60 @@ import (
 	"strings"
 
 	"github.com/golang/glog"
-	apiutil "k8s.io/kubernetes/pkg/api/util"
+	"k8s.io/kubernetes/pkg/api/unversioned"
 )
 
 // List of registered API versions.
 // The list is in the order of most preferred to the least.
-var RegisteredVersions []string
+var RegisteredGroupVersions []unversioned.GroupVersion
 
 func init() {
-	validGroupVersions := map[string]bool{
-		"v1":                       true,
-		"extensions/v1beta1":       true,
-		"componentconfig/v1alpha1": true,
-		"metrics/v1alpha1":         true,
+	validGroupVersions := map[unversioned.GroupVersion]bool{
+		unversioned.GroupVersion{Group: "", Version: "v1"}:                      true,
+		unversioned.GroupVersion{Group: "extensions", Version: "v1beta1"}:       true,
+		unversioned.GroupVersion{Group: "componentconfig", Version: "v1alpha1"}: true,
+		unversioned.GroupVersion{Group: "metrics", Version: "v1alpha1"}:         true,
 	}
 
 	// The default list of supported api versions, in order of most preferred to the least.
-	defaultSupportedVersions := "v1,extensions/v1beta1,componentconfig/v1alpha1"
```

Comment:
old default didn't include metrics, new one does... not sure if that was a bug before